tests/test_encoder.py

```diff
@@ -373,11 +384,11 @@ def test_encode_property_list(self):
 
     def test_encode_multiple_values_test(self):
         geometry = "POINT(0 0)"
-        properties1 = dict(foo="bar", baz="bar")
-        properties2 = dict(quux="morx", baz="bar")
+        properties1 = {"foo": "bar", "baz": "bar"}
+        properties2 = {"quux": "morx", "baz": "bar"}
```

Comment:
I personally preferred this style for creating dicts where keys are strings, but I'd rather be consistent with the latest style guidance. Is this being driven by ruff?

Reply:
Ruff gives an explanation [here](https://docs.astral.sh/ruff/rules/unnecessary-collection-call/): 

> It's unnecessary to call, e.g., dict() as opposed to using an empty literal ({}). The former is slower because the name dict must be looked up in the global scope in case it has been rebound.

I have never seen this rule being deactivated in open-source projects using `ruff`.

Reply:
sounds good, thanks for the explanation